tests/test_cli.py

```diff
@@ -33,6 +33,12 @@
 
 
 class CliTest():
+    def tearDown(self):
+        # Clean up global config
```

Comment:
Not sure if there's a better way to handle this, I was a bit surprised `nose` didn't handle it but I've admittedly never dealt with something like this

Reply:
Seems good enough :man_shrugging: I'd like to move to `pytest` at some point soon in any case.